src/Sylius/Bundle/CoreBundle/Controller/ProductController.php

```diff
@@ -283,4 +332,51 @@ private function renderResults(
 
         return $this->handleView($view);
     }
+
+    /**
+     * @param ArchetypeInterface $archetype
+     * @param Pagerfanta $results
+     * @param string $template
+     * @param int $page
+     * @param null $facets
+     * @param null $facetTags
+     * @param null $filters
+     * @param null $searchTerm
+     * @param null $searchParam
+     * @param null $requestMethod
```

Comment:
You should describe these parameters with theirs type and optionally write ``null`` after pipeline, for example
``@param string|null $requestMethod``.